client/src/components/EventForm/index.jsx

```diff
@@ -269,35 +241,43 @@ class EventForm extends Component {
   }
 
   renderButtons() {
-    if ('id' in this.props.match.params) return (
-      <React.Fragment>
-        <Col className="left-align" s={6}>
-          <Button waves="light" onClick={this.updateEvent}>Update Event</Button>
-        </Col>
-        <Col className="right-align" s={6}>
-          <Button
-            className="red" waves="light" onClick={() => window.$('#eventClose').modal('open')}>
-            Close Event
-          </Button>
-        </Col>
-        <Modal
-          id='eventClose'
-          header='Modal Header'
-          actions={
-            <div>
-              <Button flat modal="close" waves="light" onClick={this.closeEventHandler}>
-                Yes
-              </Button>
-              <Button flat modal="close" waves="light">
-                No
-              </Button>
-            </div>
-          }>
-          Do you really want to close this event?
-        </Modal>
-      </React.Fragment>
-    )
-    else return (
+    if ('id' in this.props.match.params) {
+      if(this.state.status_str === 'Close') return (
+        <React.Fragment>
+          <Col className="center-align" s={12}>
+            <Button waves="light" onClick={this.updateEvent}>Update Event</Button>
+          </Col>
+        </React.Fragment>
+      )
+      else return (
+        <React.Fragment>
+          <Col className="left-align" s={6}>
+            <Button waves="light" onClick={this.updateEvent}>Update Event</Button>
+          </Col>
+          <Col className="right-align" s={6}>
+            <Button
+              className="red" waves="light" onClick={() => window.$('#eventClose').modal('open')}>
+              Close Event
+            </Button>
+          </Col>
+          <Modal
+            id='eventClose'
+            header='Close Event'
+            actions={
+              <div>
+                <Button flat modal="close" waves="light" onClick={this.closeEventHandler}>
+                  Yes
```

Comment:
I think better write for button's text are "Close" and "Cancel".

Reply:
Why?

Reply:
Because most sites don't have that. That must be sounds better than "yes" or "not"